index.js

```diff
@@ -1,62 +1,42 @@
 /* eslint-disable no-unused-vars */
+
+import { Book } from './modules/book-class.js';
+import { clearFields } from './modules/clearFields.js';
+import { createListOfBooks } from './modules/showBooks.js';
+import { DateTime } from './modules/luxon.min.js';
+
 if (localStorage.getItem('My Books') === null) {
   localStorage.setItem('My Books', JSON.stringify([]));
 }
 
 const Localstoragebook = JSON.parse(localStorage.getItem('My Books'));
 
-function updateLocalStorage() {
+const updateLocalStorage = () => {
   localStorage.setItem('My Books', JSON.stringify(Localstoragebook));
-}
+};
 
-function createListOfBooks(arr) {
-  let books = '';
-  for (let i = 0; i < arr.length; i += 1) {
-    let liClass = 'dark-bakcground';
-    if (i % 2 === 0) {
-      liClass = 'book-li';
-    }
-    books += `
-              <li class= '${liClass}'>${arr[i].title} by ${arr[i].author} <button onclick="removeBook(${i})">Remove</button></li> <br />
-              `;
-  }
-  return books;
-}
-
-function showBooks() {
+const showBooks = () => {
   const listOfBooks = document.querySelector('.container');
   listOfBooks.innerHTML = `
-                <ul class="book-ul"/>
-                ${createListOfBooks(Localstoragebook)}</ul>
-            `;
-}
-
-class Book {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-  }
-}
-
-function clearFields() {
-  document.querySelector('.book-title').value = '';
-  document.querySelector('.author-name').value = '';
-}
+                  <ul class="book-ul"/>
+                  ${createListOfBooks(Localstoragebook)}</ul>
+              `;
+};
 
-function addNewBook(bookTitle, bookAuthor) {
+const addNewBook = (bookTitle, bookAuthor) => {
   const myBook = new Book(bookTitle, bookAuthor);
   Localstoragebook.push(myBook);
   updateLocalStorage();
   showBooks();
   clearFields();
-}
+};
 
-function removeBook(i) {
+const removeBook = (i) => {
```

Comment:
- [ ] Kindly fix the `Remove` book button cause now when I click on it I get this error.
![image](https://user-images.githubusercontent.com/24703171/193564537-744ab42d-9365-4c63-bb9e-cbdb81207f2e.png)